test/Interop/Cxx/reference/Inputs/reference.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TEST_INTEROP_CXX_REFERENCE_INPUTS_REFERENCE_H
+#define TEST_INTEROP_CXX_REFERENCE_INPUTS_REFERENCE_H
+
+int getStaticInt();
+int &getStaticIntRef();
+const int &getConstStaticIntRef();
+
+void setStaticInt(int);
+void setStaticIntRef(int &);
+
+auto getFuncRef() -> int (&)();
```

Comment:
Please also add tests for rvalue references. I think your implementation should just work.

Reply:
Done.